easybuild/easyconfigs/m/mrcfile/mrcfile-1.4.3-foss-2022b.eb

```diff
@@ -0,0 +1,40 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2022/11
+easyblock = 'PythonPackage'
+name = 'mrcfile'
+version = '1.4.3'
+homepage = 'https://github.com/ccpem/mrcfile'
+description = """mrcfile is a Python implementation of the MRC2014 file format, which is used in 
+structural biology to store image and volume data.
+
+It allows MRC files to be created and opened easily using a very simple API,    
+which exposes the file’s header and data as numpy arrays. The code runs in      
+Python 2 and 3 and is fully unit-tested.
+
+This library aims to allow users and developers to read and write standard-     
+compliant MRC files in Python as easily as possible, and with no dependencies on
+any compiled libraries except numpy. You can use it interactively to inspect    
+files, correct headers and so on, or in scripts and larger software packages to 
+provide basic MRC file I/O functions. """
+
+
+toolchain = {'name': 'foss', 'version': '2022b'}
```

Comment:
I see you initially used gfbf here, and I do think that makes more sense. Or was there a particular reason (e.g. dependency on something that's not part of gfbf but is part of foss) that made you switch to foss?

Reply:
Oh yes, you are right, I think I misread the deps of SciPy-bundle... I'll correct it